Practica real con tabla/ViewController.swift

```diff
@@ -149,7 +160,12 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
     @IBAction func Okqlq(sender: AnyObject)
     {
         if textfield.text != "" {
-            messages.addObject(textfield.text!)
+            messages.append(
+                Message(
+                    content: textfield.text!,
+                    isOutgoing: swiche.on
```

Comment:
If the message is sent when the switch is turned on, then mark it as a message sent by me.